### PR TITLE
Revert "adding some wise words (#5)"

### PR DESCRIPTION
**Pull request by cndystf**
This reverts commit b649f187d442bab68e0d19d2c9f45d3a9ae626a5.

### CODE REVIEW

File: logbook.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  {
+    "log": "Greetings, mermaid!",
+    "by": "ednagealmira"
+  },
+  {
+    "log": "weki weki weki weki it's time for school",
+    "by": "cndy.s"
+  }
+]
```
Comment (LordAldi) [inline logbook.json:18]: jgn dihapus lagi yaa 😢

**Review by LordAldi — APPROVED** [review]
😿

**Review by ednagealmira — COMMENTED** [review]
lgtm

File: logbook.json (hunk 1 above)
Comment (cndystf) [reply logbook.json:18]: Maap 🤣😅

**Pull request merged**